src/samples/Uno.Themes.Samples/Uno.Themes.Samples.iOS/Info.plist

```diff
@@ -45,9 +45,6 @@
 	<array>
 		<string>Fonts/uno-fluentui-assets.ttf</string>
```

Comment:
Do we still need ui-fluentui-assets.ttf to be included still?

Reply:
Ah I didn't add the Uno.Fonts.Fluent font package to replace this but I guess it makes sense to do both fonts for this PR